packages/cardano-services-client/src/HandleProvider/KoraLabsHandleProvider.ts

```diff
@@ -28,6 +29,20 @@ export interface KoraLabsHandleProviderDeps {
   policyId: Cardano.PolicyId;
 }
 
+export interface HandleResolution {
+  policyId: Cardano.PolicyId;
+  handle: Handle;
+  resolvedAddresses: {
+    cardano: Cardano.PaymentAddress;
+    hasDatum: boolean;
+    defaultForStakeKey?: Handle;
+    defaultForPaymentKey?: Handle;
+  };
+  resolvedAt: Point;
```

Comment:
```suggestion
```
The KoraLabs API doesn't return this, and the field is being made optional in #772.

Reply:
We also decided the drop the `resolvedAddresses` property, and introduce it as `otherAddresses` once there's a need, since this type strictly depends on it having a Cardano resolution.

``` ts
export interface HandleResolution {
  policyId: Cardano.PolicyId;
  handle: Handle;
  cardanoAddress: Cardano.PaymentAddress;
  hasDatum: boolean;
  defaultForStakeKey?: Handle;
  defaultForPaymentKey?: Handle;
  backgroundImage?: Asset.Uri;
  profilePic?: Asset.Uri;
}
```

Reply:
Fixed on 0938543